8.x/core-concepts/cruds.md

```diff
@@ -2,4 +2,760 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+In this tutorial, [Laravel 8](https://laravel.com) crud will be shown. And an example of crud operation can be seen in Larammerce.
+In this tutorial, you will learn crud with Laravel version 8 in Laravel. And you will get to know the concept of (create, read, update and delete).
+So several steps must be followed to get the initial raw stuff for the crud operation using controller, model, path, bootstrap, and blade. Also, in Larammerce,
+with the help of the tools developed by the Larammerce team, the path of crud operation will be much smoother, so it is necessary to follow this guideline in this tutorial.
```

Comment:
This document reviews how to run CRUD (create, read, update,  delete) in Larammerce.
As you know, CRUD is not only related to Larammerce and is related to the structure of the Laravel framework.
In larammerce, tools have been created to make CRUD easier. Also, the created CRUD should be the same as the previous CRUD of the system, so it should be followed the guidelines provided in this document.